### PR TITLE
Added Io.readBytesFromResources() to read binary data.

### PR DESCRIPTION
**Pull request by tfenne**

### CODE REVIEW

File: src/main/scala/com/fulcrumgenomics/commons/io/Io.scala [hunk 1]
```diff
@@ -191,17 +200,27 @@ trait IoUtil {
...
-      .flatMap(m => Option(m(name)))
-      .headOption match {
-        case None     =>
-          throw new IllegalArgumentException(s"Resource does not exist at path: $name")
-        case Some(in) =>
-          val stream = new BufferedInputStream(in, bufferSize)
-          Source.fromInputStream(stream).withClose(() => stream.safelyClose()).getLines
-    }
+    val stream = new BufferedInputStream(streamFromResource(name), bufferSize)
+    Source.fromInputStream(stream).withClose(() => stream.safelyClose()).getLines
```
Comment (nh13) [inline src/main/scala/com/fulcrumgenomics/commons/io/Io.scala:205]: Thanks for this too!

File: src/main/scala/com/fulcrumgenomics/commons/io/Io.scala [hunk 2]
```diff
@@ -27,6 +27,8 @@ import java.io._
 import java.nio.file.{Files, Path}
 
 import com.fulcrumgenomics.commons.CommonsDef._
+
+import scala.collection.mutable.ArrayBuffer
```
Comment (nh13) [inline src/main/scala/com/fulcrumgenomics/commons/io/Io.scala]: unused import

**Review by nh13 — APPROVED** [review]
One unneeded import, otherwise LGTM.

File: src/main/scala/com/fulcrumgenomics/commons/io/Io.scala (hunk 2 above)
Comment (tfenne) [reply src/main/scala/com/fulcrumgenomics/commons/io/Io.scala]: Doh, thanks!

**Pull request merged**